### ISSUE TITLE
Transport Tycoon (1994)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#5466]
This is for the original version, not the Deluxe version we also have in the collection.

Discord user Corsedonk noted that the tutorials dont work. @SmilingSpectre found it was due to files not named correctly. To fix rename all the `DEMOxEA.DAT` files renamed to `DEMOxAA.DAT`

He also found a copy of the combined Tycoon and Worlds expansion CD release at https://www.old-games.ru/game/6744.html which might be worth adding\fresh installing from

**Comment by SmilingSpectre** [comment]
It's the same as our rip, so no reinstall required, I think.

**Comment by exoscoriae** [comment]
I've repacked the game.

It no longer whas one install for the World Pack and one for the regular, which means there is now a user note explaining how to switch the graph sets within the game.

**Comment by Python-Exoproject** [comment]
Interesting, there is no Music in the Sound Canvas selection. Not sure if this was the case in the previous setup.

I tested using General Midi from the game installer instead and that worked so not sure why the SC55 driver isnt working. Maybe it didnt work before either and we were using the GM option previously as well?

**Comment by parricc** [comment]
@Python-Exoproject The previous setup did work with SC-55. This is a problem with the new one.

**Comment by Python-Exoproject** [comment]
I know that the exodos option for SC55 worked, but my suspicion is that behind that exodos option we may have been using the General Midi option from the games installer. Pretty much same same so GM is a fine option to pick as a fix for the problem now occurring, but if we can confirm that previously we were using the game installers SC55 option then something strange has happened

**Comment by Python-Exoproject** [comment]
Ok I dug up my kids portable with exodos 6 on it to see what it was using before we updated the game in WIP and confirmed my suspicions.

The Sound Canvas driver is buggered, its a zero kb file (music.com) and we used the General Midi game install selection for the sound canvas option, which as I said above is perfectly fine and we have done that for many games with stuffed or no SC55 specific driver.

In case this ever comes up again I will also note here that we use the GM driver for Transport Tycoon Deluxe as well for the same reason.
